src/_shared/scss/_adverts-books.scss

```diff
@@ -73,6 +73,44 @@
         }
     }
 
+    &.advert--prominent-true > .advert__image-container {
+        flex-basis: calc(66.67% - #{$gs-gutter / 2});
+    }
+
+    &.has-no-flex {
```

Comment:
Yeah sorry, I really need to prepare that CSS workshop 😩 

Another piece of knowledge: `has-no-flex` is a **feature detection class**.  When the page load, a small script tests whether the browser supports feature X and if not, adds a class `has-no-X` on the `html` element. It means the browser does not support the `flex-*` rules and with it we can provide a workaround (here, setting a `width`).